smithy-aws-iam-traits/src/main/resources/META-INF/smithy/aws.iam.json

```diff
@@ -264,6 +264,18 @@
                 "smithy.api#private": {},
                 "smithy.api#documentation": "An IAM policy principal type."
             }
+        },
+        "aws.iam#actionName": {
+            "type": "string",
+            "member": {
+                "target": "aws.iam#IamIdentifier"
+            },
+            "traits": {
+                "smithy.api#trait": {
+                    "selector": ":test(operation, member)"
```

Comment:
Sounds like this should be `"operation"`, given the documentation. Tests should be updated to reflect this as well.

Reply:
done